components/gitpod-db/src/typeorm/migration/1637586292251-IndexTokenEntryExpiryDate.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import {MigrationInterface, QueryRunner} from "typeorm";
+import { indexExists } from "./helper/helper";
+
+export class IndexTokenEntryExpiryDate1637586292251 implements MigrationInterface {
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        const TABLE_NAME = "d_b_token_entry";
+        const INDEX_NAME = "ind_expiryDate";
```

Comment:
not sure if this is necessary, `TokenGarbageCollector` is running once every 5 minutes (in theory, because it's a fixed `setInterval` again)

but, it won't do any harm.

Reply:
I also think it's not super-critical. But mind that this is used on delete:
```
EXPLAIN UPDATE d_b_token_entry AS te
SET te.deleted = TRUE
WHERE te.expiryDate != ''
AND te.refreshable != 1
AND te.expiryDate <= '...';
```
And we are doing a full table scan without this :roll_eyes: